lib/behaviour.flow

```diff
@@ -63,6 +63,14 @@ export {
 	// Retrieves the leaks
 	getSuspectCallStacks(categoryTag : ?) -> [string];
 
```

Comment:
We should add a helper function instead of exporting this. We do not want more to inspect it, so better to add a function.

Reply:
At first I want to add a getter function, but then I found, that other field of DebuggedSubscriber is also use. So if already 2 fields are used, I thought, maybe it is a sign that this stuct should bu exported?